base_plugins/rstudio_server/rstudio_server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This custom script only works on images where rstudio server is pre-installed on the Docker image
+#
+# This custom script has been tested to work on the following docker images:
+#  - jiata/aztk-r:0.1.0-spark2.2.0-r3.4.1
+#  - jiata/aztk-r:0.1.0-spark2.1.0-r3.4.1
+#  - jiata/aztk-r:0.1.0-spark1.6.3-r3.4.1
+
+if  [ "$IS_MASTER" = "1" ]; then
+
+    ## Download and install Rstudio Server
+    wget https://download2.rstudio.org/rstudio-server-$RSTUDIO_SERVER_VERSION-amd64.deb
```

Comment:
@brnleehng do you think we can move this `$RSTUDIO_SERVER_VERSION` environment out of the container and make it a plugin argument?

Reply:
I think this is fine. It will have to be done this way once we move plugins to separate containers.